modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -82,6 +82,11 @@ func (k Keeper) newQueryHandler(ctx sdk.Context, callerID string) *queryHandler
 	return newQueryHandler(ctx, k.GetQueryPlugins(), callerID)
 }
 
+// storeWasmCode stores the contract to the VM, pins the checksum in the VM's in memory cache and stores the checksum
+// in store. The checksum identifying it is returned if successful. The following checks are made to the
```

Comment:
maybe this helps it be a bit more explicit since we're dealing with the VM data directory storage, the in memory cache and 08-wasm's kvstore

```suggestion
// in the 08-wasm store. The checksum identifying it is returned if successful. The following checks are made to the
```